src/AasxServerStandardBib/AasxHttpContextHelper.cs

```diff
@@ -3635,7 +3635,10 @@ public static void securityInit()
                             var sm = env.AasEnv.FindSubmodel(smr);
                             if (sm != null && sm.idShort != null)
                             {
-                                sm.SetAllParents();
+                                if (!sm.idShort.ToLower().Contains("Security"))
```

Comment:
I didn't check if `String.Contains()` has been overridden in this application but unless that's the case, this check doesn't work. It compares a string with a capital letter against a fully lower-cased idShort.